packages/shared_preferences/shared_preferences_macos/example/android/app/src/main/java/io/flutter/plugins/sharedpreferencesexample/MainActivityTest.java

```diff
@@ -0,0 +1,15 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugins.sharedpreferencesexample;
```

Comment:
With the latest cherry-pick (v1.12.13+hotfix.4), I'm not sure we need Android files.

https://github.com/flutter/flutter/pull/46632

Reply:
I think that's for dummy android apps inside shared_preferences_macos/android, but this is the android part of the example app. But yeah, we have that here too so I can remove it now.

Done